scripts/generate_cheatsheet.go

```diff
@@ -40,6 +40,12 @@ func main() {
 
 		bindingSections := getBindingSections(mApp)
 		content := formatSections(mApp, bindingSections)
+		content = fmt.Sprintf(
+			"_This file is auto-generated. To update, make the changes in the "+
+				"pkg/i18n directory and then run `%s` from the project root._\n\n%s",
+			commandToRun(),
+			content,
+		)
```

Comment:
I followed `lazygit`'s pattern, but probably a slightly cleaner solution would be to convert `go run scripts/generate_cheatsheet.go` into a `const`. Let me know if you agree and i can update the PR.

Reply:
@gusandrioli I'm happy for that to be converted to a const

Reply:
Updated!